openreview/conference/templates/programchairWebfield.js

```diff
@@ -1049,7 +1049,7 @@ var buildPaperTableRow = function(note, reviewerIds, completedReviews, metaRevie
         note: reviewObj.id,
         rating: reviewObj.rating,
         confidence: reviewObj.confidence,
-        reviewLength: reviewObj.content.review.length,
+        reviewLength: reviewObj.content && reviewObj.content.review.length,
```

Comment:
I think this line should be `reviewObj.content.review && reviewObj.content.review.length`
We know the content object exists, but want to check that review is there.

Reply:
oops, sorry, you are right